ui/package-lock.json

```diff
@@ -0,0 +1,18726 @@
+{
```

Comment:
Remove `package-lock.json`. We already use `yarn.lock` to lock the dependencies.

```bash
# installs dependencies
yarn install

# add a dependency
yarn add <package-name>

# remove a dependency
yarn remove <package-name>
```